cpp/src/arrow/memory_pool.h

```diff
@@ -109,6 +157,14 @@ class ARROW_EXPORT MemoryPool {
 
  protected:
   MemoryPool() = default;
+
+  /// Free a memory region allocated by GetImmutableZeros().
+  ///
+  /// @param buffer Pointer to the start of the allocated memory region
+  /// @param size Allocated size located at buffer. An allocator implementation
```

Comment:
nit, but most docstrings use \param syntax

Reply:
Oops, I think my editor is configured to autocomplete using `@param` and I just didn't notice. a551146